src/elf/mod.rs

```diff
@@ -346,7 +369,7 @@ if_sylvan! {
         }
     }
 
-    fn gnu_hash_len(bytes: &[u8], offset: usize, ctx: Ctx) -> error::Result<usize> {
+    pub fn gnu_hash_len(bytes: &[u8], offset: usize, ctx: Ctx) -> error::Result<usize> {
```

Comment:
why does this need to be `pub` now? I'd like to not increase api scope unless absolutely necessary